examples/widgets/slider_demo1.py

```diff
@@ -0,0 +1,55 @@
+"""
+===========
+Slider Demo
+===========
+
```

Comment:
There is no description added, why changes in the whole file?

Reply:
Sorry about that. I think I din't save the changes. I will fix it.